io/net/virt-net/network_virtualization.py

```diff
@@ -148,6 +148,10 @@ def setUp(self):
                'failover' in str(self.name.name):
                 self.cancel("this test is not needed")
         self.local = LocalHost()
+        cmd = "echo 'module ibmvnic +p; func send_subcrq -p' > /sys/kernel/debug/dynamic_debug/control"
```

Comment:
@smruti77 is this -p or +p ?

Reply:
@narasimhan-v  this is +p only.

Reply:
@smruti77 I see -p in the second command

Reply:
@narasimhan-v  +p is for enable and -p is for disable.

Reply:
@smruti77 please check and confirm the command once. You are disabling here instead of enabling.

Reply:
@narasimhan-v no here i kept enable only.
```
enable
 echo "module ibmvnic +p; func send_subcrq -p" > /sys/kernel/debug/dynamic_debug/control
disable
 echo "module ibmvnic -p; func send_subcrq -p" > /sys/kernel/debug/dynamic_debug/control
```

Reply:
I did check on my env,  the commands look fine for enable and disable

Reply:
"func send_subcrq -p" is the syntax to trigger dynamic debug in debugfs control file
module ibmvnic +p/-p    enables/disables the debug data to stdout

I did see loads of messages for every vnic operation when +p 
and very minimal messages when -p